src/modules/copyFile.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs');
+
+const copyFile = (sourcePath, destPath) => {
+  if (sourcePath === destPath) {
+    return;
+  }
+
+  try {
+    const data = fs.readFileSync(sourcePath, 'utf-8');
+
+    fs.writeFileSync(destPath, data, 'utf-8');
+  } catch (error) {
+    throw new Error('Something went wrong: ', error);
```

Comment:
Why throw new Error? At least save the message. Also the second param [is wrong](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error)

```suggestion
    throw new Error('Something went wrong: ' + error.message);
```